### DISCUSSION TITLE
Configurare mail da parte del registro

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#150] [Richieste di aiuto] [answered]
Salve, come posso configurare le mail da inviare per le credenziali del registro?

**Comment by trinko** [comment]
Se intendi l'indirizzo a cui inviare l'email, da utente amministratore nella configurazione degli utenti.

Se intendi il contenuto, devi modificare i template delle credenziali presenti nelle cartelle:
- **templates/email**
- **templatest/pdf**
-

**Comment by manuelbarberi** [comment]
Intendo la configurazione per l’invio credenziali mail da parte del
registro

Il giorno lun 31 gen 2022 alle 10:02 Antonello Dessì <
***@***.***> ha scritto:

**Reply by sempliciniluigi89** [reply, marked as answer]
Anche la mia scuola avrebbe bisogno , se è possibile ,lo chiedo io ,come tutte le scuole ,un piccolo video di tutta la configurazione per l'invio delle credenziali da parte del registro.

**Comment by trinko** [comment]
La configurazione per l'uso delle email si trova nel file **.env**, nella riga che contiene "MAILER_DSN"

Se si usa un account gmail per la spedizione, la configurazione sarà la seguente:
`
MAILER_DSN=gmail://UTENTE:PASSWORD@default?verify_peer=0
`
dove:
- **UTENTE** conterrà l'utente gmail (es. se la mail è _pippo@gmail.com_, l'utente è _pippo_)
- **PASSWORD** conterrà la password dell'utente indicato
Il resto va lasciato invariato.

Sono possibili altre configurazioni, ma dipendono dal servizio di hosting usato e da cosa permette.

**Reply by fabriziotoppi** [reply]
dove si trova il file .env?

**Comment by manuelbarberi** [comment]
Con quella configurazione non funziona. Errore: potrebbe essere un problema
al server di posta elettronica
Il giorno gio 3 feb 2022 alle 19:06 Antonello Dessì <
***@***.***> ha scritto:

**Comment by trinko** [comment]
Può essere dovuto alla presenza di caratteri particolari nella PASSWORD.
Fai attenzione che non ci siano:
- lettere accentate
- i caratteri speciali:  ":", "@", "/", "?"

Inoltre, controlla che sia configurat l'indirizzo email per le notifiche: come utente amministratore, menu SCUOLA > ISTITUTO

**Comment by manuelbarberi** [comment]
Ho provato a configurare in questo modo ma non va

Il giorno dom 6 feb 2022 alle 11:11 Antonello Dessì <
***@***.***> ha scritto:

**Comment by sempliciniluigi89** [comment]
Anche io ho provato ,ma niente

Il Dom 6 Feb 2022, 16:08 Manu ***@***.***> ha scritto:

**Comment by manuelbarberi** [comment]
Le sto tentando tutte ma niente

Il giorno dom 6 feb 2022 alle 16:37 sempliciniluigi89 <
***@***.***> ha scritto:

**Comment by zerai** [comment]
Eventualmente per chi avesse problemi con l'invio mail tramite google
c'è un opzione da attivare nel proprio account  'Less secure app' che solitamente è da attivare (leggere bene la doc)

https://support.google.com/accounts/answer/6010255

**Reply by fabriziotoppi** [reply]
non si puo piu fare la google dal 30 maggio 22

**Reply by zerai** [reply]
https://support.google.com/accounts/answer/185833

**Reply by fabriziotoppi** [reply]
ma te sei riuscito?

**Comment by sempliciniluigi89** [comment]
Sarebbe disposto ad aiutarmi in questo per favore?

Il Dom 6 Feb 2022, 18:38 zero ***@***.***> ha scritto:

**Comment by manuelbarberi** [comment]
Già provato ad attivare ma niente lo stesso
Il giorno dom 6 feb 2022 alle 18:57 sempliciniluigi89 <
***@***.***> ha scritto:

**Reply by fabriziotoppi** [reply]
sei risucito  alla fine a configuare la mail?

**Reply by manuelbarberi** [reply]
Google ha bloccato le connessioni esterne!

**Comment by fabriziotoppi** [comment]
non va... ho creato la pass ma non va

Il giorno gio 15 giu 2023 alle ore 15:28 zero ***@***.***> ha
scritto:

**Comment by fabriziotoppi** [comment]
qulacuno c'e riuscito?

**Reply by zerai** [reply]
https://github.com/symfony/symfony-docs/issues/17115

**Comment by trinko** [comment]
Dipende tutto dal servizio di hosting che state usando: se utilizzate l'hosting free **altervista** l'unico sistema funzionante è quello indicato come **"server locale tramite PHP"**. Gli altri sono bloccati da loro.

**Reply by fabriziotoppi** [reply]
funziona alla grande. grazie mille
